go.mod

```diff
@@ -0,0 +1,3 @@
+module easemesh-api
```

Comment:
Why not full qulified repository url?

Reply:
According to Istio's `module istio.io/api`, is that `model megaease/easemesh-api` qulfied enough?

Reply:
I will update it to `module github.com/megaease/easemesh-api` according to our EG project's module.